Source/Foundation/bsfUtility/Prerequisites/BsRTTIPrerequisites.h

```diff
@@ -162,15 +162,17 @@ namespace bs
 	 *
 	 * @see		RTTIPlainType<T>
 	 */
-#define BS_ALLOW_MEMCPY_SERIALIZATION(type)					\
-	template<> struct RTTIPlainType<type>					\
-	{	enum { id=0 }; enum { hasDynamicSize = 0 };			\
-		static void toMemory(const type& data, char* memory)		\
-		{ memcpy(memory, &data, sizeof(type)); }			\
-		static UINT32 fromMemory(type& data, char* memory)		\
+#define BS_ALLOW_MEMCPY_SERIALIZATION(type)								\
+	static_assert (std::is_trivially_copyable<type>()==true,			\
+						#type " is not trivially copyable");			\
```

Comment:
Is there a way to fix Github's display of tabstops so that they match the project's tabstop value?

It's really annoying that the way they're displayed on github are so different than in someone's text editor.

Reply:
Well,  we have `.editorconfig`, but I don't think github reads those

Reply:
I think GitHub used to read `.editorconfig` at some point and tabs were displayed fine. Not sure what changed.